src/ast/irbuilderbpf.cpp

```diff
@@ -603,62 +617,73 @@ CallInst *IRBuilderBPF::CreateGetRandom()
 
 CallInst *IRBuilderBPF::CreateGetStackId(Value *ctx, bool ustack, StackType stack_type)
 {
+  assert(ctx && ctx->getType() == getInt8PtrTy());
   assert(bpftrace_.stackid_maps_.count(stack_type) == 1);
-  Value *map_ptr = CreateBpfPseudoCall(bpftrace_.stackid_maps_[stack_type]->mapfd_);
+
+  Value *mapfd = CreateBpfPseudoCall(
```

Comment:
```suggestion
  Value *map_ptr = CreateBpfPseudoCall(
```
From what I understand BpfPseudo is converting our file descriptor into a pointer to the actual map struct, so the original variable name makes more sense